certlogic/certlogic-js/src/evaluator.ts

```diff
@@ -66,6 +66,16 @@ const compare = (operator: ComparisonOperator, values: Comparable[]): boolean =>
     }
 }
 
+type DateTimeComparisonOperator = "after" | "before" | "not-after" | "not-before"
+const comparisonOperatorForDateTimeComparison = (operator: DateTimeComparisonOperator) => {
+    switch (operator) {
+        case "after": return ">"
+        case "before": return "<"
+        case "not-after": return "<="
+        case "not-before": return ">="
+    }
+}
+
 const evaluateBinOp = (operator: string, values: CertLogicExpression[], data: any): any => {
```

Comment:
Just a side note. Looks like this function is about evaluating not only `binary operation` but also `ternary` and etc. Maybe makes sense to call e.g. `evaluateExpression`

Reply:
Renamed to `evaluateInfix` because “infix” is more specific than “expression”, and seems to fit the bill.

Reply:
I like it